websites/B/bol.com/presence.ts

```diff
@@ -0,0 +1,37 @@
+const presence = new Presence({
+  clientId: "813110347165728849"
+});
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "bol",
+    buttons: [{ label: "Pagina bekijken", url: document.location.href }]
```

Comment:
if it will not redirect to mainpage sure 🤷 
![image](https://user-images.githubusercontent.com/9348108/108813395-3db70d80-75b1-11eb-816b-6d32408468fd.png)